modules/gapi/perf/common/gapi_render_perf_tests.cpp

```diff
@@ -0,0 +1,9 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2020 Intel Corporation
+
+
+#include "../perf_precomp.hpp"
```

Comment:
I think, there isn't any reason for create seperate *.hpp file for render perfomance tests

Reply:
You can move to *.cpp file

Reply:
I think I did not understand you correctly
Could you clarify your point

Reply:
You can move all stuff from this file to corresponding cpp file

Reply:
Is now its ok?